### DISCUSSION TITLE
ImageConversion

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#3169] [Help]
How to convert tiff, wmf ,emf image format to png 

Any MagickWand method to conevert tiff,wmf ,emf image to png format ?

**Comment by snibgo** [comment]
Read the input file with MagickReadImage(), then write the image with MagickWriteImage().

**Reply by programmerarunkumar** [reply]
How to add emf image format to ImageMagick in ubuntu environment?

**Comment by snibgo** [comment]
Perhaps your IM can already read EMF. Try:
```
magick -list format |grep -i emf

      EMF  EMF       r--   Windows Enhanced Meta File
      WMF  EMF       r--   Windows Meta File
```
If you can't already read EMF, perhaps config.log can tell you why. Look for WMF as well as EMF.
